.github/workflows/hil_tests.yml

```
@@ -126,7 +126,7 @@ jobs:
           - hil_board: esp32_devkitc_wrover
             west_board: esp32_devkitc_wrover/esp32/procpu
             manifest: .ci-west-zephyr.yml
-            extra_twister_args: '--flash-before'
+            extra_twister_args: '--flash-before --west-flash=--esp-tool=`which esptool.py`,--esp-device=${!PORT_VAR}'
```

```suggestion
            extra_twister_args: '--flash-before --west-flash=--esp-tool=$(which esptool.py),--esp-device=${!PORT_VAR}'
```
I see `$(command)` syntax getting more and more popular compared to backticks. After using it for a while I find it more readable, easier to maintain and more future-proof.

See https://mywiki.wooledge.org/BashFAQ/082